pentasharp/Controllers/AuthenticateController.cs

```diff
@@ -128,5 +130,39 @@ public IActionResult DeleteConfirmed(int id)
             }
             return RedirectToAction("UserList");
         }
+
+        public IActionResult Login()
+        {
+            return View(new LoginViewModel());
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Login(LoginViewModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View(model);
+            }
+
+            var user = await _context.Users.SingleOrDefaultAsync(u => u.Email == model.Email);
+            if (user == null || user.PasswordHash != HashPassword(model.Password))
+            {
+                ModelState.AddModelError(string.Empty, "Email or password are incorrect.");
```

Comment:
would it be better to implement a standardized response structure, so we are more consistent and the project would be easier to maintain. What i mean by this is that we would create an APIStatusEnum that would look like:
  public enum ApiStatusEnum
  {
      OK = 0, //Success
      USER_NOT_FOUND = 1
}

Create a StandardResponse that would have the statusEnum, UUID, and a message as parameters
return a JSON, for e.g.         
return Json(new StandardResponse(ApiStatusEnum.USER_NOT_FOUND, model.Email, "User not found."));

also for traceability use loggers, a typical logger looks like: 
_logger.LogWarning("Login failed for user {Email}. Error: {ErrorCode} - {ErrorMessage}", model.Email, ApiStatusEnum.USER_NOT_FOUND, "User not found.");

so if we would have a tracing platform linked to our project, all the logs would be found there